src/commands/dev/dev.ts

```diff
@@ -271,6 +272,7 @@ export const createDevCommand = (program: BaseCommand) => {
     .option('-d ,--dir <path>', 'dir with static files')
     .option('-f ,--functions <folder>', 'specify a functions folder to serve')
     .option('-o ,--offline', 'disables any features that require network access')
+    .option('--disable-edge-functions', "disables edge functions. use this if your environment doesn't support Deno")
```

Comment:
I don't think we should treat this as a customer-facing flag. The only use case we have for it is purely internal, and we haven't fully considered the experience people will get if they disable edge functions. I suggest we use `.hideHelp()` here ([example](https://github.com/netlify/cli/blob/edbfbb452de28e7aa31d27c4dd125112e64ece17/src/commands/dev/dev.ts#L265)) so it's removed from the docs, and I would even consider renaming it to something like `--internal-disable-edge-functions`. That way we're not committing to indefinitely supporting something we're just exploring at this point.

Reply:
that's a very fair point. implemented in https://github.com/netlify/cli/pull/6495/commits/6540b878e458e26d15aac1c2c0e7c1edf6d5cab8